e2e/cypress/integration/enterprise/elasticsearch_autocomplete/renaming_spec.js

```diff
@@ -129,7 +129,7 @@ describe('Autocomplete with Elasticsearch - Renaming', () => {
             });
 
             // # Rename the team
-            cy.apiPatchTeam(testTeam.id, {name: 'updatedteam' + timestamp});
+            cy.apiPatchTeam(testTeam.id, {display_name: 'updatedteam' + timestamp});
```

Comment:
Team "name" is actually not allowed to be patched. Only the those included at https://github.com/mattermost/mattermost-webapp/compare/master...saturninoabril:MM-26075-e2e_standardize_api_commands?expand=1#diff-aa488add4a43ce7a79a0dadab2523052R57-R63 are allowed.